core/contracts/Voting.sol

```diff
@@ -268,30 +266,8 @@ contract Voting is Testable, Ownable, OracleInterface, VotingInterface, Encrypte
         migratedAddress = newVotingAddress;
     }
 
-    /**
-     * @notice Adds the provided identifier as a supported identifier. Price requests using this identifier will be
-     * succeed after this call.
-     */
-    function addSupportedIdentifier(bytes32 identifier) external onlyOwner {
-        if (!supportedIdentifiers[identifier]) {
-            supportedIdentifiers[identifier] = true;
-            emit SupportedIdentifierAdded(identifier);
-        }
-    }
-
-    /**
-     * @notice Removes the identifier from the whitelist. Price requests using this identifier will no longer succeed
-     * after this call.
-     */
-    function removeSupportedIdentifier(bytes32 identifier) external onlyOwner {
-        if (supportedIdentifiers[identifier]) {
-            supportedIdentifiers[identifier] = false;
-            emit SupportedIdentifierRemoved(identifier);
-        }
-    }
-
     function isIdentifierSupported(bytes32 identifier) external view returns (bool) {
```

Comment:
Do we still need this method? Shouldn't external callers be calling the `IdentifierWhitelist` contract directly?

Reply:
I kept this here because the method `isIdentifierSupported` is in the `OracleInterface`, but per our discussion I will remove it